codecov_auth/commands/owner/interactors/update_default_organization.py

```diff
@@ -30,7 +31,10 @@ def update_default_organization(self, default_org: Optional[Owner]):
             owner_id=self.current_owner.ownerid
         )
         owner_profile.default_org = default_org
-        return owner_profile.save()
+        saved_owner_profile = owner_profile.save()
+        if default_org:
```

Comment:
We should be able to get rid of this line since it's accounted for above in the `validate` function

Reply:
We're not raising any exceptions in the validation function if the org is `None` so it looks like we can still execute this flow with a null org.